remote/blueprints/image.py

```diff
@@ -111,24 +112,70 @@ def fit(box: int):
 
     # Now we have a valid image.
     # Fit it into a box of the specified size.
-    _image: Image = handler.image
-    _format = _image.format and _image.format.lower()
+    image = handler.image
 
     start = time.time()
-    _resized_content = _rescale_aspect_ratio(_image, box)
-    if _resized_content is None:
+    if handler.animated and request.args.get("animated"):
```

Comment:
Seems here has changed the signature, I think it's better let @livid  know.

Reply:
It is a required feature to turn on animated image processing or not.